apps/mobile/src/features/AccountsSheet/MyAccounts/hooks/useMyAccountsSortable.ts

```diff
@@ -13,24 +12,13 @@ export const useMyAccountsSortable = (): useMyAccountsSortableReturn => {
   const dispatch = useDispatch()
   const safes = useSelector(selectAllSafes)
   const [sortableSafes, setSortableSafes] = useState(() => Object.values(safes))
-  const isEdit = useSelector(selectMyAccountsMode)
-
-  useEffect(() => {
-    const newSafes = Object.values(safes)
```

Comment:
Did you check the drag and drop of the sortable when removing this effect? I remember we were having a issue after sorting a wallet... That is why I created this useEffect 🤔

Reply:
Yes, I did. As you can see few lines later I moved the setSortableSafes to the onDragEnd callback. 

I removed the shouldGoToListMode as now it is possible to remove a safe even if it is the only one in the app.